docs/src/components/ExampleSnippet/ExampleSnippet.tsx

```diff
@@ -24,6 +24,13 @@ const ExampleSnippet = ({ render = () => null, value }: ExampleSnippetProps) =>
   const string =
     value ||
     reactElementToJSXString(element, {
+      displayName: (element: React.ReactElement<any>): string =>
```

Comment:
https://github.com/algolia/react-element-to-jsx-string#reactelementtojsxstringreactelement-options

Reply:
Original function uses `No Display Name`, that breaks `prettier.parser.babylon`

https://github.com/algolia/react-element-to-jsx-string/blob/0b17eaf3decafac13f276e705d4c850418cd4bb7/src/parser/parseReactElement.js#L19